java/client/src/main/java/glide/api/models/commands/stream/StreamReadGroupOptions.java

```diff
@@ -100,4 +103,39 @@ public String[] toArgs(String group, String consumer, Map<String, String> stream
 
         return optionArgs.toArray(new String[0]);
     }
+
+    /**
+     * Converts options and the key-to-id input for {@link StreamBaseCommands#xreadgroupBinary(Map,
+     * GlideString, GlideString, StreamReadGroupOptions)} into a GlideString[].
+     *
+     * @return GlideString[]
+     */
+    public GlideString[] toArgsBinary(
```

Comment:
Do we need this if we have https://github.com/aws/glide-for-redis/pull/1782?

Reply:
Not sure how #1782 solves the issue here since we receive GS arguments to the method here and on the other PR we have no args.